possel/application.py

```diff
@@ -82,17 +82,19 @@ def get_etc_file(filename):
 
 def get_arg_parser():
```

Comment:
It would be easier to put them into the default config of the configuration object, as right now if a value is set from argparse (including a default) it would override anything else lower down with the same name